README.md

```diff
@@ -14,33 +14,156 @@ The system uses a number of open source projects to work properly:
 * [ElizaBOT-JS] - Javascript implementation of the ELIZA specification by Weizenbaum, 1966. Special thanks to Landsteiner!
 
 
-# Installation:
+## Installation:
 
-TEXA requires [Go Lang](https://golang.org/)  v1.7+ to run.
+### Go:
 
-Install the dependencies, devDependencies and start the server.
+TEXA is written in [Go](http://golang.org) v1.7+. If you don't have a Go
+development environment, please [set one up](http://golang.org/doc/code.html).
+
+### Redis: 
+
+Texa using the [texalib]
+to store the slabs, CatPages and MtsPages, in [redis](https://redis.io/)
+[Redis quick start](https://redis.io/topics/quickstart#redis-quick-start).
+
+### MongoDB:
+
+TEXA using [MongoDB](https://docs.mongodb.com/manual/installation) to persist the conversation between 
```

Comment:
Again, suggesting to reform the sentence as:

"TEXA persists all your chat history in a local MongoDB instance, keeping your interrogation data private. You can install [MongoDB](https://docs.mongodb.com/manual/installation) and learn more about MongoDB [here](https://docs.mongodb.com/manual/tutorial/getting-started/)."